### DISCUSSION TITLE
2023-07-05

### DISCUSSION THREAD

**Discussion by woo-jk** [woowacourse-teams/2023-haru-study#7] [daily_meeting]
### 개인 일정
- 노아: 5시 근로

### 어제의 회고
P:
- 기능에 대한 의견이 아닌 진행 방식에 대한 의견으로 시간을 많이 소비한 것이 아쉬웠다.
- 하루종일 기획, 기술 관련 이야기만 한 것 같아서 지치는 면도 있다!
- 회의 시간이 50분이 넘어가는 경우가 있다
- 회의 중 의견 제시에 대해 약간 부담이 있다.
- 중간 중간 논의에서 힘 빠지는 순간에 잘 대처하지 못한 것 같다.

T:
- 진행자에게 확실한 권한을 주면 어떨까?
- 환기가 필요할 때 누군가 무중력 광장에서 이야기하자고 제안해보자.
- 중간중간에 잡담을 5분씩이라도 섞어보면 어떨까요?
- 타이머 활용
- 회의 중 의견 제시에 더 편한 분위기가 조성되면 좋을 듯하다.
- 개발과 전혀 상관 없는 잡담을 한다. (오늘 정말 효과가 좋았다!)

### 액션아이템 선정

회의 진행자가 잡담 시간 선언하기
타이머로 회의 시간 확실히 50분 한정하기
할 일 Issue 탭에 정리해 할 사람 Assign하기(프로젝트 관련 기능이 정리된 이후)

### 오늘의 할 일

- 공통
  - 도메인 용어 사전 완성하기 (4시)
- 프론트엔드
  - 웹팩 설정
  - wiki 기술 스택(styled-component 선정 이유) 정리
  - 와이어 프레임 스케치 마무리 조금하기
- 백엔드
  - 개발문서 마무리(기술 스택 및 선정 이유 정리)
  - API 명세

### 내일의 할 일

- 공통
  - 발표 준비
  - 그라운드 룰 wiki 고도화 및 팀 문화 정리
  - 로고 및 서비스 메인 색상 얘기해보기
  - github Project 칸반보드 정리
  - 기술블로그 세팅
- 백엔드
  - 기술문서 마무리(자바 뭐쓸지, 스프링 뭐쓸지)
- 프론트엔드
  - 와이어 프레임 제작

**Comment by jaehee329** [comment]
### 체크인
9점 - 많이 잤는데 엄청 개운한 느낌은 아니에요. 근데 비 오고 나니 훨 시원해서 좋네

### 체크아웃
9점 - 눈이 피곤한데 할 건 많이 했네요

### KPT
K: 
P: 전체 회의가 갈 수록 줄어들텐데 회의록을 어떻게 할 지 정해야 될 것 같다.
T:

**Comment by aak2075** [comment]
### 체크인

9점 - 선선한 날씨, 쿠폰으로 공짜 커피를 사서 매우 좋으나 출근길 지하철이 지연되어서(-1)

### 체크아웃

9점 - 오늘도 열심히 회의를 했다.

### KPT

K: 중간에 잡담 타임을 가지는 것이 좋았다.
P: 회의를 할 때 그냥 각자 의견만 주장하는 느낌이 든다. 그러다 보니 서로가 한 이야기를 제대로 이해를 하지 못하는 경우가 있다. 정말 좋은 의견인데 묻혀 넘어가버린다면? 과연 우리 끼리도 이해하지 못했는데 다른 사람이 들었을 때 이해할 수 있을까?
T: 우선 서로의 의견에 대해 주의 깊게 듣는 태도가 필요할 것 같다. 그리고 나서 이해가 안간다면 얼마든지 되물어 볼 수 있어야 하지 않을까?

**Reply by woosung1223** [reply]
마코 왜 h2로 함 ㅡㅡ

**Reply by aak2075** [reply]
하지만 빨랐죠

**Comment by woosung1223** [comment]
### 체크인
9점 - 카페라떼가 맛있어서 9점

### 체크아웃
7점 - API 설계가 생각보다 어려웠어서 7점임니두

### KPT
K: 점심 한식뷔페 먹은 게 좋았습니다. 가끔 한번씩 가도 좋을거같다!
P: 토론하는 동안 나오는 이야기들을 기록해야 할 필요성이 있는 것 같다. 자꾸 어디까지 이야기했었는지 까먹게 된다.
T: 회의하다가 공동으로 한 중요한 의사결정이 있다면, 기록해두고 넘어가기? 세이브포인트처럼.

**Comment by woo-jk** [comment]
### 체크인
10점 - 날씨가 시원해서 좋다. 피곤하지도 않다 ㅎㅎ

### 체크아웃
10점 - 웹팩 설정과 초기 세팅을 모두 마쳐서 기분이 좋다

### KPT
K: 잡담시간 있는거 좋다.
P: 오늘은 딱히 문제점이 없었다.
T: 없다.

**Comment by nlom0218** [comment]
### 체크인

10점 - 상쾌한 아침 🌅

### 체크아웃

10점 - 빌드 공부와 테스트 공부가 아주 맛깔났다

### KPT

K: 오류를 해결하기 위한 과정은 항상 가져갔으면 한다!
P: 회의록에 어떤 것을 적을지? 다함께 의논할 내용에 대해서만 다룰지?가 명확하지 않다. 개인적인 의견은 스프린트를 시작할 때, 중요한 전체 안건이 있을 때, 적으면 어떨까?
T: 회의록!!!

**Comment by yeopto** [comment]
### 체크인

10점 - 오늘 굉장히 시원하다.

### 체크아웃

10점 - 오늘 기본 개발 환경 셋팅을 마쳐서 뿌듯하다.

### KPT

K: 각자 업무를 분담해서 병렬적으로 일을 하는 것이 좋았다.
P: 없었다.
T: 없다.

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 오늘 드디어 기획 이외의 주제로 회의를 한다는 것에 기쁘지만 어제 일찍 잠들지 못한 스스로 아쉬워서 -1점 입니당.

### 체크아웃
8점 - 생각보다 API명세가 쉽지 않아서 오래걸린게 아쉽습니다...!

### KPT
K: 액션 아이템을 적재적소에 잘 쓰고 있다.
P: 할 일 목록이 실시간으로 정리할 곳이 없다.
    논의과정에서 발생한 문제상황과 이를 해결하기 위해 모색한 과정들이 잘 기록되고 있는지 의문이 든다.
T: 먼저 할 일 목록 관리 페이지를 만들어뒀다가 프로젝트 사용이 가능해지면 이슈로 옮긴다.
    문제상황과 모색한 해결방안, 해결방안 선정 이유를 최대한 회의록에 정리하고 기술블로그에 정제해서 포스팅한다.
